src/doc/commands/commands_ini.nit

```diff
@@ -0,0 +1,361 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module commands_ini
+
+import doc::commands::commands_model
+
+# Cmd that finds the ini file related to an `mentity`
+abstract class CmdIni
+	super CmdEntity
+
+	# Ini file
+	var ini: nullable ConfigTree = null
+
+	redef fun init_command do
+		var res = super
+		if not res isa CmdSuccess then return res
+		var mentity = self.mentity.as(not null)
+
+		if not mentity isa MPackage then return new WarningNoIni(mentity)
+
+		var ini = mentity.ini
+		if ini == null then return new WarningNoIni(mentity)
+
+		self.ini = ini
+
+		return res
+	end
+end
+
+# No ini file for `mentity`
+class WarningNoIni
+	super CmdWarning
+
+	# MEntity provided
+	var mentity: MEntity
+
+	redef fun to_s do return "No ini file for `{mentity.full_name}`"
+end
+
+# Cmd that finds the ini description related to an `mentity`
+class CmdIniDescription
+	super CmdIni
+
+	# Ini description
+	var desc: nullable String = null
+
+	redef fun init_command do
+		var res = super
+		if not res isa CmdSuccess then return res
+
+		var mentity = self.mentity.as(not null)
+		var ini = self.ini.as(not null)
+
+		var desc = ini["package.desc"]
+		if desc == null then return new WarningNoDescription(mentity)
+		self.desc = desc.trim
```

Comment:
unrelated but I'm asking myself if `lib/ini` should not auto-trim the values

Reply:
In fact, the keys and values are already trimmed. You can remove them

https://github.com/nitlang/nit/blob/a46817d351713e3877ada8c482e83c0d74a60ac5/lib/ini/ini.nit#L216